docs/visualization/wcsaxes/ticks_labels_grid.rst

```diff
@@ -161,7 +161,7 @@ The syntax for the format string is the following:
 ``'%d'``              ``'15'``
 ==================== ====================
 
-All the ``h...``, ``d...``, ``m...``, and ``s...`` formats can be used for
+All the ``h...``, ``d...``, ``m...``, and ``s...`` formats should be used for
```

Comment:
So to just check, I can not and will never be able to use "x.xxx" for a angular coordinate axes?